src/components/RsvpBadge/RsvpBadge.js

```diff
@@ -5,24 +5,33 @@ const toK = (n) => {
   return n >= 1000 ? (n / 1000).toFixed(1) + 'K' : n;
 };
 
-const RsvpBadge = ({ totalAccepted, center }) => {
-  const alignStyle = center ? styles.wrapperCenter : styles.wrapperRight;
-
+const RsvpBadge = ({ totalAccepted, center, type }) => {
+  const alignStyle = center ? styles.wrapperCenter : styles.wrapperRight,
+    spanRSVP = <span className={styles.shortText}>{totalAccepted}+ Going</span>
```

Comment:
Might be more readable and consistent w/our other variable declarations to declare this separately, like:
```
const alignStyle = center ? styles.wrapperCenter : styles.wrapperRight;
const spanRSVP = <span className={styles.shortText}>{totalAccepted}+ Going</span>;
```

Whaddyafink?

Reply:
Sure.